spec/system/products_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe "Products", type: :system do
+  let(:taxon) { create(:taxon) }
+  let(:product) { create(:product, name: "test", taxons:[taxon]) }
+  let!(:related_product) { create(:product, name:"test", taxons: [taxon]) }
+  
+  before do
+    related_product.images << create(:image) 
+    product.images << create(:image)
+    visit potepan_product_path(id: product.id)
+  end
+
+  it "ページタイトルが正常か" do
+    expect(page).to have_title "#{product.name} - BIGBAG Store"
+  end
+    
+  it "商品情報が取得できているか" do
+    expect(page).to have_content product.name
+    expect(page).to have_content product.description
+    expect(page).to have_content product.display_price
+  end
+    
+  it "HOMEを押したらtopページにいけること" do
+    within(".breadcrumb") do
+      click_on "Home"
+      expect(page).to have_current_path potepan_path
+    end
+  end
+
+  it "一覧ページへ戻るを押したらその商品ページにいけること" do
+    product.master.images.create(attachment: create(:image).attachment)
+    click_on "一覧ページへ戻る"
+    expect(current_path).to eq potepan_category_path(taxon.id)
+  end
+
+  it "関連商品の情報が取得できているか" do
+    expect(page).to have_content related_product.name
+    expect(page).to have_content related_product.display_price.to_s
+  end
+    
+  it "関連商品をクリックしたらページにいけること" do
+    page.first(".related_product").click
```

Comment:
可能ならクラス名からDOMを特定するよりは、ページ内に実際に表示されている文言を使ってclickをエミュレートしたほうがより堅牢なテストになります。例えば:
```suggestion
    click_on related_product.name
```